docs/index.rst

```diff
@@ -25,6 +25,13 @@
 
    <a class="github-fork-ribbon right-top fixed" href="https://github.com/scikit-hep/pyhf/" data-ribbon="View me on GitHub" title="View me on GitHub">View me on GitHub</a>
 
+
+.. raw:: html
+
+   <p id="dev-version"><strong>Warning:</strong> This is a development version. The latest stable version is at <a href="https://pyhf.readthedocs.io/">ReadTheDocs</a>.</p>
+
+   <p id="old-version"><strong>Warning:</strong> This is not a stable version. The latest stable version is <a id="latest-version-link" href="https://pyhf.readthedocs.io/">v?.?.?</a>.</p>
```

Comment:
```suggestion
   <p id="old-version"><strong>Warning:</strong> This is not the latest stable version. The latest stable version is <a id="latest-version-link" href="https://pyhf.readthedocs.io/">v?.?.?</a>.</p>
```

I think I'd still call old releases stable, but we just want to encourage using newer releases. So maybe just mention it is not the latest stable?